_posts/2016-01-20-release-add-ons-1.8.md

```diff
@@ -0,0 +1,129 @@
+---
+title:  "Release: Add-Ons 1.8"
+---
+
+We are proud to announce the availability of Add-On Self Administration 1.8 and Add-On Administration 1.8.
+This release mostly brings compatibility fixes to make the add-ons work with all versions of OSIAM >= 2.2 again.
```

Comment:
But 2.2 is the lower bound now, because the `ADMIN` scope only exists since 2.2. So, before 1.7, the add-on was compatible with virtually any OSIAM version. 1.7 was only compatible with OSIAM 2.3, which was a bug. 1.8 restores support for OSIAM versions >= 2.2.